src/scenes/Projects/Overview/ProjectOverview.graphql

```diff
@@ -76,6 +76,12 @@ fragment ProjectOverview on Project {
   partnerships {
     ...PartnershipSummary
   }
+
+  ... on Postable {
```

Comment:
You shouldn't need to specify this. Only maybe I just fixed that in develop schema last night

Reply:
Let's actually move the comments list to another component & query. This will
- Not further add to the clutter of ProjectOverview
- Allows us to use the `useListQuery` hook which List wants and allows for pagination
- Data can load in parallel

Reply:
I'm still not able to get it to work without `... on Postable` - I just merged latest updates from develop today.

Reply:
Moved the post list into a separate component and query 👍